Data Structures/Linked List/C++/linkedList.cpp

```diff
@@ -0,0 +1,77 @@
+#include <bits/stdc++.h>
```

Comment:
Now, this is a stray file. You may want to delete this @anuchak

Reply:
Done!